src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -39,6 +39,10 @@ const ApplicationModal = ({
     setHideForm(true);
   };
 
+  const handleReject = () => {
+    //need to add a post request to the mentee_intake and mentor_intake database to update the validateStatus from pending to rejected.
```

Comment:
potentially could be better to leave this function and comment out and just describe as a ticket in trello to reduce repo bloat

Reply:
I've got that cleaned up, and will update the trello card to include this when it's ready! Thank you!